src/sidebar/store/modules/annotations.ts

```diff
@@ -567,6 +573,37 @@ const savedAnnotations = createSelector(
   annotations => annotations.filter(ann => isSaved(ann)) as SavedAnnotation[],
 );
 
+/**
+ * Return the list of unique users who authored any annotation, ordered by username.
+ */
+const usersWhoAnnotated = createSelector(
+  (state: State) => state.annotations,
+  annotations => {
+    const usersMap = new Map<
+      string,
+      { user: string; username: string; displayName: string | null }
+    >();
+    annotations.forEach(anno => {
+      const { user } = anno;
+      const username = getUsername(user);
+      const displayName = anno.user_info?.display_name ?? null;
+
+      // Keep a unique list of users
+      if (!usersMap.has(user)) {
+        usersMap.set(user, { user, username, displayName });
+      }
+    });
+
+    // Sort users by username
+    return [...usersMap.values()].sort((a, b) => {
+      const lowerAUsername = a.username.toLowerCase();
+      const lowerBUsername = b.username.toLowerCase();
+
+      return lowerAUsername.localeCompare(lowerBUsername);
```

Comment:
I note that `localeCompare` does accept a third argument that allows specifying the sensitivity to case and accents - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#browser_compatibility. In theory that should be faster as it avoids creating many copies of each input string, but you've have to measure.

Reply:
using `sensitivity: base` also has the benefit that we don't need to convert to lowercase first. I'll use it.